lib/transport_web/controllers/session_controller.ex

```diff
@@ -7,8 +7,10 @@ defmodule TransportWeb.SessionController do
   alias Transport.Datagouvfr.{Authentication, Client.User}
   require Logger
 
+  @user_attrs {"id", "apikey", "email", "first_name", "last_name"}
```

Comment:
Why not make it a list, so you don't need Tuple.to_list after.

Reply:
Right